isis/src/mro/apps/hi2isis/MroHiriseInstrument.trn

```diff
@@ -701,22 +701,22 @@ End_Group
 
 Group = SpiderLeg120Temperature
```

Comment:
Although it is not required, the group names should be changed to keep confusion down.

Reply:
Good catch. Groups names now updated.